wandb/sdk/data_types/table.py

```diff
@@ -640,16 +730,33 @@ def to_json(self, run_or_artifact):
         json_dict = super().to_json(run_or_artifact)
 
         if isinstance(run_or_artifact, wandb.wandb_sdk.wandb_run.Run):
+            _type = "table-file"
```

Comment:
required: don't use a leading underscore when naming a local variable (call this `wbvalue_type` or something)

nit: move to an `else` block so that it's only assigned once